goat.sh

```diff
@@ -7,6 +7,16 @@ errcho(){
     >&2 echo $@
 }
 
+# Check for $EDITOR and editor in the path
+if [ -n "$EDITOR" ]; then
+    EDITOR_COMMAND="$EDITOR"
+elif command -v editor >/dev/null 2>&1; then
+    EDITOR_COMMAND="editor"
+else
+    EDITOR_COMMAND=""
```

Comment:
Actually, I think I would just change the environmental variable `EDITOR` to `vi` and secondly set `vi` as a fallback editor if nothing is expicitly specified.